indexing/src/main/java/com/google/enterprise/cloudsearch/sdk/indexing/IndexingItemBuilder.java

```diff
@@ -428,6 +429,20 @@ public IndexingItemBuilder setSearchQualityMetadataQuality(FieldOrValue<Double>
     return this;
   }
 
+  /**
+   * Set the {@code contextAttributes} field value for the {@code ItemMetadata}.
+   * <p>
+   *     Using this setter will override the default value for contextAttributes field which is empty.
+   * </p>
+   *
+   * @param contextAttributes the {@code contextAttributes} instance
+   * @return this instance
+   */
+  public IndexingItemBuilder setContextAttributes(List<ContextAttribute> contextAttributes) {
```

Comment:
Can we add/modify E2E integration test to make sure that this change works?

Reply:
I have added a basic integration test which indexes 2 items with context attributes and checks if the item is searchable. I have also manually checked that context attributes are getting populated for these items. When I send a manual search request with context attributes I am seeing the feature working as expected.